types/index.ts

```diff
@@ -52,7 +52,7 @@ export interface ProjectPayload {
   }
   overview?: Block[]
   site?: string
-  slug: string
+  slug?: string
```

Comment:
Hi, thank you for this PR! 👋 

Why do you feel that it would make sense to have the project slug be optional in the type?
[In the schema](https://github.com/sanity-io/template-nextjs-personal-website/blob/main/schemas/documents/project.ts#L28) we define the slug as a mandatory item 🙂